section 1: outline of the course and diagnostic images overview

drcandacemakedamoore opened the pull request:
This had a flaw, but I think I just fixed it The point of the PR is the general intro lessons (not the MRI lesson which I am still working on)

gcroci2 commented:
Is the PR ready to be reviewed? I see that you assigned me to it, but you're still pushing commits @drcandacemakedamoore

gcroci2 commented on episodes/introduction_to_medical_imaging.md:
```diff
@@ -65,102 +55,74 @@ associated with the lessons. They appear in the "Instructor View"
-- Run `sandpaper::build_lesson()` to preview your lesson locally
+- Every type and sequence of imaging gives you different and more information 
+- Computationally images have arrays but arrays are surrounded by other data
+- Research should be designed with human limitations in mind
```
Not clear what this refers to in the introductory text. I'd reformulate the concept a bit, or make it more explicit in the text above

gcroci2 commented on episodes/introduction_to_medical_imaging.md:
```diff
@@ -65,102 +55,74 @@ associated with the lessons. They appear in the "Instructor View"
+- Every type and sequence of imaging gives you different and more information 
+- Computationally images have arrays but arrays are surrounded by other data
+- Research should be designed with human limitations in mind
+- We can expect more imaging modalities in the future
```
Is anything about new possible imaging modalities mentioned in the text? I'd either mention it, or remove this key point

gcroci2 reviewed (changes requested):
Nice great start of the new course :D I am so excited about it!

I have a comment about the general structure of this "Introduction to Medical Imaging". The objectives are very well centered, but I think that the text structure after the "OVERVIEW" tab can be polished a bit.
1. Referring to the first objective ("Explain various common kinds of medical images"), in the episode text I'd highlight more the main types of diagnostic images (very few but clear and punctual sentences for each technology or technique), like listing them in bullet points (or dedicating a short subsection to each technique), and specifying which ones will be covered in more depth in the lesson's episodes (e.g., MRI). In the current file, there are some more information in the subsections about x-rays and CT only. I'd stick more to a complete list with some very basic info (or the same but in the form of subsection, as I mentioned above). We can also have a chat about this if you think it can help!
2. Referring to the the second objective ("Understand how data for them was created and is organized in these images at a high level"), I'd create a subsection about diagnostic images storage or something along these lines, and I'd move there the parts in which you mention the DICOM format, the 2D-arrays, etc. and I'd make a bit more explicit the explanation about what _metadata_ are in the diagnostic imaging context, mentioning the word metadata and its meaning (e.g., patient info, example of patient info). I'd keep everything very short, no need to dig into anything, but useful to mention.

I added some minor comments and suggestions across the text, and I have few notes about unnecessary files in the current PR:
- please remove the episodes/introduction_to_mri.md, that does not belong here :)
- some figures are not actually used in the .md files, so I'd suggest to remove them for now and just add them when will be actually needed: bids_app.png, bids_structure.jpg, T2w.gif.
- same for the figures used in the mri lesson, I'd remove them from this PR: bold.gif, coordinate_systems.png, dwi_tracts.png, dwi.gif, fmri_timeseries.png, mri_slices.jpg, numpy_arrays.png, t1t2flairbrain.jpg, T1w.gif.

Finally, I saw that by mistake you've already pushed an earlier version on the intro markdown file to main, so to avoid that in the future I inserted rules for the repo that will prevent us doing this, i.e., we can't push to main unless a PR is open and unless another collaborator approves it. This helped me a lot in the past in avoiding confusion with the other developers and in avoiding pushing stuff by mistake. 

Just ask me a new review when you'll be done with integrating this :)

drcandacemakedamoore commented on episodes/introduction.md:
```diff
@@ -1,39 +1,46 @@
 
-What you need to know is that there are three sections required for a valid
-Carpentries lesson:
+The interpretation of medical images for clinical purposes involves skills that take
```
OK, will change, good point.

drcandacemakedamoore commented on episodes/introduction.md:
```diff
@@ -1,39 +1,46 @@
-[R Markdown][r-markdown] for dynamic files that can render code into output. 
-Please refer to the [Introduction to The Carpentries 
-Workbench](https://carpentries.github.io/sandpaper-docs/) for full documentation.
+This is a lesson created in the style of The Carpentries. It is written with the assumption
```
Will do.

drcandacemakedamoore commented on episodes/introduction.md:
```diff
@@ -1,39 +1,46 @@
-What you need to know is that there are three sections required for a valid
-Carpentries lesson:
+The interpretation of medical images for clinical purposes involves skills that take
+highly trained professionals like nuclear medicine specialists and
```
check

drcandacemakedamoore commented on episodes/introduction.md:
```diff
@@ -46,68 +53,74 @@ associated with the lessons. They appear in the "Instructor View"
-dynamic reports with {knitr}, so we now use mathjax to describe this:
+## Lectures
+
+Some of our episodes are based on presentations.
```
I'll get rid of some of it.

drcandacemakedamoore commented:
Thanks for such a thorough review. I considered all of your points, and changed a lot of things.  I removed one of the BIDS-illustrating files, but I believe the other will be used when I update the MRI section next. I realize I originally made a mistake to put the draft MRI file  (and accompanying images) there, but I think in this case we can just leave it, otherwise I risk making more of a mess. I also somehow accidentally pushed to main before a review, but now main seems protected. I can't explain exactly how this was. Anyways, awaiting a re-review @gcroci2

gcroci2 commented:
Wow thanks for the very quick integration! I'll do my best to re-review before the end of today :) One comment:

> I removed one of the BIDS-illustrating files, but I believe the other will be used when I update the MRI section next. I realize I originally made a mistake to put the draft MRI file (and accompanying images) there, but I think in this case we can just leave it, otherwise I risk making more of a mess. 

I can remove the files myself on this PR, I think this will be beneficial later when you will open the MRI one (it will be clearer for me as a reviewer which files are new). If you agree to this, I can push the commit myself :) Nothing is going to change on your other branch where you're working on the MRI episode. @drcandacemakedamoore

gcroci2 reviewed (approved):
I rendered the first two episodes using R studio and the lesson looks great!!

To make the process faster, I added a few commits myself containing the final suggestions I had, in particular https://github.com/esciencecenter-digital-skills/medical-image-processing/pull/9/commits/df4719484528b57c57418005e6b24f0df4b664a8 for `introduction.md` and https://github.com/esciencecenter-digital-skills/medical-image-processing/pull/9/commits/234698857302249b678a5a44619d9153da2fb6e8 for `introduction_to_medical_imaging.md`, mainly containing typos fixes and headers enhancements, and https://github.com/esciencecenter-digital-skills/medical-image-processing/pull/9/commits/a65de3ecb79a79cc76c23e781deaae7fe5495a7f for removing the files not related to the current PR.

Let me know if you like the changes, otherwise we can always revert them or add more edits. Otherwise, feel free to merge :) @drcandacemakedamoore
